README.md

```diff
@@ -49,6 +49,154 @@ When CheckQC starts and no path to the config file is specified it will give you
 the path to where the default file is located on your system, if you want a template
 that you can customize according to your own needs.
 
+When you run CheckQC you can expect to see output similar to this:
+
+```
+checkqc  tests/resources/170726_D00118_0303_BCB1TVANXX/
+INFO     ------------------------
+INFO     Starting checkQC (1.1.2)
+INFO     ------------------------
+INFO     Runfolder is: tests/resources/170726_D00118_0303_BCB1TVANXX/
+INFO     No config file specified, using default config from /home/MOLMED/johda411/workspace/checkQC/checkQC/default_config/config.yaml.
+INFO     Run summary
+INFO     -----------
+INFO     Instrument and reagent version: hiseq2500_rapidhighoutput_v4
+INFO     Read length: 125-125
+INFO     Enabled handlers and their config values were:
+INFO            ClusterPFHandler Error=unknown Warning=180
+INFO            Q30Handler Error=unknown Warning=80
+INFO            ErrorRateHandler Error=unknown Warning=2
+INFO            ReadsPerSampleHandler Error=90 Warning=unknown
+INFO            UndeterminedPercentageHandler Error=10 Warning=unknown
+WARNING  QC warning: Cluster PF was to low on lane 1, it was: 117.93 M
+WARNING  QC warning: Cluster PF was to low on lane 7, it was: 122.26 M
+WARNING  QC warning: Cluster PF was to low on lane 8, it was: 177.02 M
+ERROR    Fatal QC error: Number of reads for sample Sample_pq-27 was too low on lane 7, it was: 6.893 M
+ERROR    Fatal QC error: Number of reads for sample Sample_pq-28 was too low on lane 7, it was: 7.104 M
+INFO     Finished with fatal qc errors and will exit with non-zero exit status.
+```
+
+The program will summarize the type of run it has identified and output any warnings and/or errors in finds.
+If no errors were found the CheckQC will output a non-zero exit status. This means it can easily be used to
+decide if a further steps should run or not, e.g. in a workflow.
+
+In addition to the normal output CheckQC has a json mode, enabled by adding `--json` to the commandline.
+This output the results normally shown in the log as json on `stdout`, so that this can either be written to
+a file, or redirected to other programs which can parse the data further. In this example we use the python
+json tool to pretty print the json output:
```

Comment:
wouldn't it be good if the log output could either be suppressed or redirected to e.g. stderr or a file? perhaps it will mess up piping and parsing to have the json output together with the log output?

Reply:
Log output is already on `stderr` (as it should be :wink:)